src/actions/UserActions.ts

```diff
@@ -36,12 +37,14 @@ export async function updateOrganizationForUser(
   userId: string,
   organization: Organization
 ): Promise<User> {
+  const organizationVerified = await verifyOrg(organization);
```

Comment:
We should move this over to the backend, src/pages/api/user.ts:52 . Since this is all FE code, it's possible that user could modify the response from verifyOrg and verify themselves even though they had an invalid organization. Not really sure what the motive would be for this, but it's better to be on the safe side of things